frontend/packages/vkt/src/components/publicEnrollment/steps/FillContactDetails.tsx

```diff
@@ -0,0 +1,127 @@
+import { ChangeEvent, useEffect, useState } from 'react';
+import { CustomTextField, H3 } from 'shared/components';
+import { TextFieldTypes } from 'shared/enums';
+import { InputFieldUtils, StringUtils } from 'shared/utils';
+
+import { PersonDetails } from 'components/publicEnrollment/steps/PersonDetails';
+import { useCommonTranslation, usePublicTranslation } from 'configs/i18n';
+import { useAppDispatch, useAppSelector } from 'configs/redux';
+import { PublicEnrollment } from 'interfaces/publicEnrollment';
+import { updatePublicEnrollment } from 'redux/reducers/publicEnrollment';
+import { publicEnrollmentSelector } from 'redux/selectors/publicEnrollment';
+
+export const FillContactDetails = ({
+  disableNext,
+}: {
+  disableNext: (disabled: boolean) => void;
+}) => {
+  const { t } = usePublicTranslation({
+    keyPrefix: 'vkt.component.publicEnrollment.steps.fillContactDetails',
+  });
+  const translateCommon = useCommonTranslation();
+
+  const [fieldErrors, setFieldErrors] = useState({
+    email: '',
+    emailConfirmation: '',
+    phoneNumber: '',
+  });
+
+  const { enrollment } = useAppSelector(publicEnrollmentSelector);
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    const hasFieldErrors = !!(fieldErrors.email || fieldErrors.phoneNumber);
+
+    const hasBlankFieldValues = [
+      enrollment?.email,
+      enrollment?.phoneNumber,
+    ].some((v) => !v || StringUtils.isBlankString(v));
```

Comment:
`StringUtils.isBlankString(v)` ei tainnut toimia yksinään tuota `!v` tsekkausta. Sitä voisi mahdollisesti yrittää korjata ellei jätä täällä näin. Yhtenä vaihtoehtona olisi myös, että alustetussa ilmoittautumisessa (`enrollment`) olisivat `email`, `emailConfirmation` ja `phoneNumber` kentät tyyppiä `string` ja arvolla `''`. Näin taisi olla AKR:ssä tyhjät kentät contactRequestille.